name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/TradeRepublicPDFExtractor.java

```diff
@@ -1304,6 +1322,173 @@ private void addAccountStatementTransaction()
                         .wrap(TransactionItem::new));
     }
 
+    private void addAccountStatementTransaction_Format02()
+    {
+        final DocumentType type = new DocumentType("KONTO.BERSICHT", //
+                        documentContext -> documentContext //
+                                        // @formatter:off
+                                        // Erstellt am 29 Apr. 2024
+                                        // @formatter:on
+                                        .section("year") //
+                                        .match("^Erstellt am [\\d]{2} .* (?<year>[\\d]{4})$") //
+                                        .assign((ctx, v) -> ctx.put("year", v.get("year"))));
+
+        this.addDocumentTyp(type);
+
+        // @formatter:off
+        // 02 Apr. Überweisung Einzahlung akzeptiert: DE7243872432 auf 2024 DE7243872432 1.200,00 € 51.352,41 €
+        // @formatter:on
+        Block depositBlock = new Block("^[\\d]{2} [\\w]{3,4}([\\.]{1})? .berweisung Einzahlung akzeptiert.*$");
+        type.addBlock(depositBlock);
+        depositBlock.setMaxSize(1);
+        depositBlock.set(new Transaction<AccountTransaction>()
+
+                        .subject(() -> {
+                            AccountTransaction accountTransaction = new AccountTransaction();
+                            accountTransaction.setType(AccountTransaction.Type.DEPOSIT);
+                            return accountTransaction;
+                        })
+
+                        .section("date", "amount", "currency") //
+                        .documentContext("year") //
```

Comment:
what if user generates statement from Dec 2023 to Jan 2024?

Reply:
We need a pdf debug to check this.
Did you have anyone?!